src/Seq.App.Opsgenie/Classes/OpsGenieData.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+// ReSharper disable MemberCanBePrivate.Global
+// ReSharper disable AutoPropertyCanBeMadeGetOnly.Global
+// ReSharper disable UnusedAutoPropertyAccessor.Global
+
+// ReSharper disable ClassNeverInstantiated.Global
+
+namespace Seq.App.Opsgenie.Classes
```

Comment:
Organizing into `Classes` and `Interfaces` might not give as much insight into the structure of the codebase as some alternatives - could this one go in `Seq.App.Opsgenie.Api`, perhaps, since the structure of these types (presumably) matches the requirements of the Opsgenie API?

Reply:
Yeah should be fine. I'll take a look.

Reply:
Reorganised into Api and Client